wordpress-mobile/WordPress-iOS
# Fix navbar icon spacing

Fixes #5804 

Sorry about the late addition @sendhil. Nothing major however.

To test:

1. In a new post with a long site verify it doesn't run over the "Preview" button
![simulator screen shot aug 2 2016 7 51 22 pm](https://cloud.githubusercontent.com/assets/1158819/17351257/b5e6a62a-58f2-11e6-991e-056c59e4a640.png)


2. Upload a photo and verify the "Uploading..." text doesn't go under the loading indicator
![simulator screen shot aug 2 2016 7 51 29 pm](https://cloud.githubusercontent.com/assets/1158819/17351279/e006a8a6-58f2-11e6-98ec-fafe131ff51d.png)


3. Verify spacing is even in Post Preview 
![simulator screen shot aug 2 2016 8 49 52 pm](https://cloud.githubusercontent.com/assets/1158819/17351258/b5e8a772-58f2-11e6-9128-3ea9b8b008a9.png)


Needs review: @sendhil
